databricks/koalas/base.py

```diff
@@ -238,7 +244,17 @@ def __radd__(self, other):
         else:
             return column_op(Column.__radd__)(self, other)
 
-    __rsub__ = column_op(Column.__rsub__)
+    def __rsub__(self, other):
+        # Note that timestamp subtraction casts arguments to integer. This is to mimic pandas's
+        # behaviors. pandas returns 'timedelta64[ns]' from 'datetime64[ns]'s subtraction.
```

Comment:
How about make this to warning message or something so that uses recognize this difference explicitly? - Since the behavior between Koalas and pandas looks way different for me -

For example.

```python
>>> kdf1.end_date - kdf1.start_date
... Warning: Note that timestamp subtraction casts arguments to integer. This is to mimic pandas's behaviors. pandas returns 'timedelta64[ns]' from 'datetime64[ns]'s subtraction.
0    35542769
1    33641969
2    31568369
Name: end_date, dtype: int64

>>> pdf1.end_date - pdf1.start_date
0   411 days 08:59:29
1   389 days 08:59:29
2   365 days 08:59:29
dtype: timedelta64[ns]
```